lib/attributor/types/hash.rb

```diff
@@ -378,16 +385,16 @@ def self.describe(shallow=false)
 
     # TODO: Think about the format of the subcontexts to use: let's use .at(key.to_s)
     attr_reader :contents
-    
-    def_delegators :@contents, 
-      :[], 
-      :[]=, 
-      :each, 
-      :size, 
-      :keys, 
-      :key?, 
-      :values, 
-      :empty?, 
+
+    def_delegators :@contents,
```

Comment:
I wonder if it'd be good to define the methods manually, instead of using the delegator code, which is slow and allocates objects every time.
Specifically, the "[]","[]=","size","key?" and "empty?" ... those are commonly used in the fast path...